### PR TITLE
Added accepted EIPs and ordered Proposed by Progress

### PR DESCRIPTION
**Pull request by MadeofTin**
Based on the Core Dev Call notes and this spreadsheet. 
https://docs.google.com/spreadsheets/d/1Mgo7mJ6b6wimUwafsMo1l-b44uec28E_Hq8EQ7YdeEM/edit#gid=0

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1679 requires approval from one of (@5chdn, @axic)

**Comment by axic** [comment]
I think this ordering is really confusing. I cannot telly if anything was removed or anything was added, additionally not sure by what metrics is the ordering decided. Do we need that ordering or are we OK applying the decision made on the calls?

**Comment by MadeofTin** [comment]
I was asked to order it this way. But I may have misunderstood which you meant to order. What did you mean here? https://gitter.im/ethereum/AllCoreDevs?at=5d1f0c5611bfea0b67a83072

The ordering is based on this spreadsheet.
https://docs.google.com/spreadsheets/d/1Mgo7mJ6b6wimUwafsMo1l-b44uec28E_Hq8EQ7YdeEM/edit#gid=0

I am fine applying decisions made on the calls.

**Comment by MadeofTin** [comment]
I could put the total completed tasks next to the EIP number so why it is ordered that way is more clear. That is the sum of the checkpoints in the sheet.

**Comment by axic** [comment]
> I was asked to order it this way. But I may have misunderstood which you meant to order. What did you mean here? https://gitter.im/ethereum/AllCoreDevs?at=5d1f0c5611bfea0b67a83072

Right, sorry I meant to move them into the sections we have discussed: Rejected, Tentatively Agreed, etc. Though I guess there is no real agreement on most of these :(

**Comment by axic** [comment]
Looking at the EIP again, it has these sections:
- Included
- Accepted
- Rejected
- Proposed

By now I would have assumed that some are rejected and more of them moved to the accepted stage. I suppose we'll nail down what is the accepted/rejected column on the next call.

**Comment by MadeofTin** [comment]
What its the difference between included and accepted?

The list I keep track of has
Discussed
- Yes
- Yes* (Tentative)
- TBD
- No* (Tentative)
- No
----
- Not Discussed.

**Comment by axic** [comment]
"Accepted" in my mind was something people agreed on is a worthy proposal and that it should be fully developed (in clients). You could equate this to "Tentatively yes".

"Included" is what actually made it into the hardfork. You could equate this to "Yes".

I would have hope we agree on a bunch of proposed ones early on, their proposers can implement them by the second deadline and we'd really just narrow down the "Accepted" list into the "Included" list in the last 2-3 months prior the fork.

File: EIPS/eip-1679.md [hunk 1]
```diff
@@ -24,52 +24,51 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Accepted EIPs
 
-- TBD
+- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
+- [EIP-2024](https://github.com/ethereum/EIPs/pull/2129): Blake2b Precompile
```
Comment (axic) [inline EIPS/eip-1679.md]: I think there was an agreement on 1108.

Comment (MadeofTin) [reply EIPS/eip-1679.md]: I remembered something on it, but I wasn't totally sure. I can add that one in

**Comment by MadeofTin** [comment]
Included should be Implemented in that case IMHO.

And accepted should be clear that they are tentative yes.

**Comment by axic** [comment]
Should also link to the decision at least in this PR so that it is trackable if someone wants to track it, for 1702/2024 here's the decision: https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2063.md

**Comment by MadeofTin** [comment]
That is a great idea

File: EIPS/eip-1679.md (hunk 1 above)
Comment (axic) [reply EIPS/eip-1679.md]: From meeting 62:
> Hudson Jameson: EIP-1108 is approved in principle, pending a benchmark which will be reviewed at the next meeting.

**Comment by axic** [comment]
Superseded by #2214.

**Pull request closed without merging**